agent.sample.yml

```diff
@@ -0,0 +1,35 @@
+# agent.yml
+
+# We need to tell Pinger to run the agent in standalone mode since the
+# default behaviour is something else.
+standalone: true
+
+# Configuration for where metrics are stored.
+metrics:
+    backend: timescale
+    host: timescale
+    port: 5432
+    username: postgres
+    password: password
+    db_name: pinger
```

Comment:
This metrices is hardcorded in docker compose file.
There should one file which contains the configs. If i as a user make changes to it, it should get reflected everywhere. In this agent.yml, if i need to change the password or any another metrics, i will have to make changes to docker compose file as well.

Reply:
So, how can we setup a single file for it? Pinger needs the `agent.yml`. And I don't think we can parse `agent.yml` in the docker-compose. So, should I just set it such that docker-compose will check for the password and db_name also in the `.env` file? But even then, we'll have to make changes at two places, right?